flow/scripts/floorplan.tcl

```diff
@@ -101,7 +101,18 @@ if {[info exists ::env(FOOTPRINT_TCL)]} {
 }
 
 # remove buffers inserted by yosys/abc
-remove_buffers
+if { [info exists ::env(REMOVE_ABC_BUFFERS)] && $::env(REMOVE_ABC_BUFFERS) == 1 } {
+  remove_buffers
+} else {
+  set additional_args "-verbose"
+  append_env_var additional_args SETUP_SLACK_MARGIN -setup_margin 1
+  append_env_var additional_args HOLD_SLACK_MARGIN -hold_margin 1
```

Comment:
What does hold margin mean before cts? Is there a risk of over fixing for hold before the clocks are propagated?

Reply:
We're not doing any hold fixing at this step, so this option doesn't make sense.   I removed it.